lib/gossip/dissemination.js

```diff
@@ -182,7 +194,7 @@ Dissemination.prototype._issueAs = function _issueAs(filterChange, mapChanges) {
             continue;
         }
 
-        if (change.piggybackCount > this.maxPiggybackCount) {
+        if (change.piggybackCount >= this.maxPiggybackCount) {
```

Comment:
Previously the counter started at 1, because it was bumped before it hit this statement.
Now the counter is at 0 when it firstly hits this statement.